code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -318,6 +364,25 @@ Difficulty: Hard
 		return TRUE
 	return FALSE
 
+
+/mob/living/simple_animal/hostile/megafauna/bubblegum/proc/hit_up_narsi()
+	SetRecoveryTime(20)
+	visible_message("<span class='colossus'><b>[pick("[SSticker.cultdat.entity_name], I call on YOU for one of MY favours you owe me!", "[SSticker.cultdat.entity_title1], I call on you for some support...", "Let us see how you like the minions of [SSticker.cultdat.entity_title2]!", "Oh, [SSticker.cultdat.entity_title3] join me in RENDING THIS WHELP APART!")]</b></span>")
+	var/list/turfs = new/list()
+	var/constructs = 0
+	for(var/turf/T in view(6, target))
+		if(T.density)
+			continue
+		if(T in range(2, target))
+			continue
+		turfs += T
+		var/amount = enraged ? 4 : 3
+		while(constructs < amount && length(turfs))
+			var/turf/spot = pick_n_take(turfs)
```

Comment:
if theres no spot, you should probably return

Reply:
done